test/files/emails/postgrey.txt

```diff
@@ -1,12 +1,6 @@
-HELO mail.external.tld
-MAIL FROM: user@external.tld
-RCPT TO: user1@localhost.localdomain
-DATA
 From: Docker Mail Server <dockermailserver@external.tld>
 To: Existing Local User <user1@localhost.localdomain>
 Date: Sat, 22 May 2010 07:43:25 -0400
 Subject: Postgrey Test Message
-This is a test mail.
 
-.
-QUIT
+This is a test mail.
```

Comment:
Unnecessary line break introduced between subject and body? (_change is not consistent with other files_)